openmeter/productcatalog/driver/feature.go

```diff
@@ -124,7 +124,7 @@ func (h *featureHandlers) CreateFeature() CreateFeatureHandler {
 
 type (
 	ListFeaturesHandlerRequest  = feature.ListFeaturesParams
-	ListFeaturesHandlerResponse = commonhttp.Union[[]api.Feature, pagination.PagedResponse[api.Feature]]
+	ListFeaturesHandlerResponse = commonhttp.Union[[]api.Feature, pagination.Result[api.Feature]]
```

Comment:
Not related to this PR, but we should get rid of this commonhttp.Union altogether and get rid of the untyped union in the typespec in the next api revision. cc @tothandras this might worth checking while creating the new api where we use untyped unions.